### DISCUSSION TITLE
!!! SECURITY ISSUE !!!: CRYPTO SCAM USING GITHUB NOTIFICATIONS

### DISCUSSION THREAD

**Discussion by bn-l** [community/community#84707] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I've noticed an increase in crypto scams using github notifications to email people directly. I reported it with a full description but got an automated email asking for "more information". See here 👇 :

![image](https://github.com/community/community/assets/13162197/b4c28d74-fd74-433c-8fc0-18003f2f0f51)

### This is the scam. They create a comment on an issue:

![image](https://github.com/community/community/assets/13162197/69da7209-d54e-4cbb-afea-6e6a8b1a2ffc)

### And then @ mention many different usernames:

![image](https://github.com/community/community/assets/13162197/3e2ad0d6-6855-4179-972e-250c286b2eaf)

😠 

I've reported both incidences but I'm sure I'll get another email "requesting more information". The timing of this attack is smart as I can imagine people are still away.

**Comment by Akash1134** [comment, marked as answer]
Hi @bn-l , 👋🏻 We really appreciate you flagging this and we're sorry about the trouble caused. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) which I see you seem to have already done. Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by YulawOne** [reply]
There's no way to report - PR, comment in PR.  There also no shortcut to report notification without clicking on it

**Reply by bn-l** [reply]
100% a dark pattern to reduce support costs

**Reply by elvisisvan** [reply]
ironically authentic comments like these would also get flagged as spam and no doubt that would be the doings of those malicious crypto reward spam accounts
